README.md

```diff
@@ -1087,6 +1087,12 @@ IEEE Int. Conf. Computer Vision (ICCV), 2021. [PDF](https://openaccess.thecvf.co
 - <a name="Li21iccv"></a>Li, S., Feng, Y., Li, Y., Jiang, Y., Zou, C., Gao, Y.,  
 *[Event Stream Super-Resolution via Spatiotemporal Constraint Learning](https://openaccess.thecvf.com/content/ICCV2021/papers/Li_Event_Stream_Super-Resolution_via_Spatiotemporal_Constraint_Learning_ICCV_2021_paper.pdf)*,  
 IEEE Int. Conf. Computer Vision (ICCV), 2021. [Code](https://github.com/lisiqi19971013/EventStream-SR), [Dataset](https://github.com/lisiqi19971013/event-based-datasets).
+- <a name="Kai23icip"></a>Kai, D., Zhang, Y., Sun, X.,  
+*[Video Super-Resolution Via Event-Driven Temporal Alignment,](https://ieeexplore.ieee.org/abstract/document/10222922/)*,  
+IEEE International Conf. on Image Processing (ICIP), 2023. [Code](https://github.com/DachunKai/EBVSR), [PDF](https://ieeexplore.ieee.org/abstract/document/10222922/).
+- <a name="Kai24icml"></a>Kai, D., Lu, J., Zhang, Y., Sun, X.,  
+*[EvTexture: Event-driven Texture Enhancement for Video Super-Resolution,](https://arxiv.org/pdf/2406.13457v1)*,  
+International Conf. on Machine Learning (ICML), 2024. [Project](https://dachunkai.github.io/evtexture.github.io/), [Code](https://github.com/DachunKai/EvTexture), [PDF](https://arxiv.org/pdf/2406.13457v1).
```

Comment:
abbreviate International

Reply:
No need for [PDF] if it is the same link as in the title.
Also, I think you do not need the v1 in arXiv
https://arxiv.org/pdf/2406.13457 works fine